config/main.py

```diff
@@ -656,26 +659,36 @@ def load_minigraph():
     log_info("'load_minigraph' stopping services...")
     _stop_services()
 
-    config_db = ConfigDBConnector()
-    config_db.connect()
-    client = config_db.get_redis_client(config_db.CONFIG_DB)
-    client.flushdb()
-    if os.path.isfile('/etc/sonic/init_cfg.json'):
-        command = "{} -H -m -j /etc/sonic/init_cfg.json --write-to-db".format(SONIC_CFGGEN_PATH)
-    else:
-        command = "{} -H -m --write-to-db".format(SONIC_CFGGEN_PATH)
-    run_command(command, display_cmd=True)
-    client.set(config_db.INIT_INDICATOR, 1)
-    if device_type != 'MgmtToRRouter':
-        run_command('pfcwd start_default', display_cmd=True)
-    if os.path.isfile('/etc/sonic/acl.json'):
-        run_command("acl-loader update full /etc/sonic/acl.json", display_cmd=True)
-    run_command("config qos reload", display_cmd=True)
-
-    # Write latest db version string into db
-    db_migrator='/usr/bin/db_migrator.py'
-    if os.path.isfile(db_migrator) and os.access(db_migrator, os.X_OK):
-        run_command(db_migrator + ' -o set_version')
+    # For Single Asic platform the namespace list has the empty string
+    # for mulit Asic platform the empty string to generate the config
+    # for host
+    namespace_list = ['']
+    if sonic_device_util.get_num_npus() > 1:
+        namespace_list += sonic_device_util.get_namespaces()
+
+    for namespace in namespace_list:
+        if namespace is '':
+            config_db = ConfigDBConnector()
+            cfggen_namespace_option = " "
+            ns_cmd_prefix = " "
+        else:
+            config_db = ConfigDBConnector(use_unix_socket_path=True, namespace=namespace)
+            cfggen_namespace_option = " -n {}".format(namespace)
+            ns_cmd_prefix = "sudo ip netns exec {}".format(namespace)
+        config_db.connect()
+        client = config_db.get_redis_client(config_db.CONFIG_DB)
+        client.flushdb()
+        if os.path.isfile('/etc/sonic/init_cfg.json'):
+            command = "{} -H -m -j /etc/sonic/init_cfg.json {} --write-to-db".format(SONIC_CFGGEN_PATH, cfggen_namespace_option)
+        else:
+            command = "{} -H -m --write-to-db {} ".format(SONIC_CFGGEN_PATH,cfggen_namespace_option)
+        run_command(command, display_cmd=True)
+        client.set(config_db.INIT_INDICATOR, 1)
+        if device_type != 'MgmtToRRouter':
+            run_command('{} pfcwd start_default'.format(ns_cmd_prefix), display_cmd=True)
+        if os.path.isfile('/etc/sonic/acl.json'):
+            run_command("acl-loader update full /etc/sonic/acl.json", display_cmd=True)
+        run_command("{} config qos reload".format(ns_cmd_prefix), display_cmd=True)
```

Comment:
687-691 do not apply to default namespace in multi-asic environment.

Reply:
Fixed in the latest commit